draft-ietf-httpbis-priority.md

```diff
@@ -108,101 +114,100 @@ The term control stream is used to describe the HTTP/2 stream with identifier
+
+## Disabling RFC 7540 Priorities {#disabling}
+
+The problems and insights set out above provided the motivation for deprecating
+RFC 7540 stream priority (see {{Section 5.3 of RFC7540}}).
+
+The SETTINGS_DEPRECATE_RFC7540_PRIORITIES setting is defined by this document in
+order to allow endpoints to explicitly opt out of using HTTP/2 priority signals
+(see {{Section 5.3.2 of HTTP2}}). Endpoints are expected to use an alternative,
+such as the scheme defined in this specification.
+
+The value of SETTINGS_DEPRECATE_RFC7540_PRIORITIES MUST be 0 or 1. Any value
+other than 0 or 1 MUST be treated as a connection error (see {{Section 5.4.1 of
+HTTP2}}) of type PROTOCOL_ERROR.
 
 Endpoints MUST send this SETTINGS parameter as part of the first SETTINGS frame.
-A sender MUST NOT change the SETTINGS_DEPRECATE_HTTP2_PRIORITIES parameter value
+A sender MUST NOT change the SETTINGS_DEPRECATE_RFC7540_PRIORITIES parameter value
 after the first SETTINGS frame. Detection of a change by a receiver MUST be
 treated as a connection error of type PROTOCOL_ERROR.
 
 Until the client receives the SETTINGS frame from the server, the client SHOULD
-send the signals of the HTTP/2 priority scheme  (see {{motivation}}) and the
-signals of this prioritization scheme (see {{header-field}} and
-{{h2-update-frame}}). When the client receives the first SETTINGS frame that
-contains the SETTINGS_DEPRECATE_HTTP2_PRIORITIES parameter with value of 1, it
-SHOULD stop sending the HTTP/2 priority signals. If the value was 0 or if the
-settings parameter was absent, it SHOULD stop sending PRIORITY_UPDATE frames
-({{h2-update-frame}}), but MAY continue sending the Priority header field
-({{header-field}}), as it is an end-to-end signal that might be useful to nodes
-behind the server that the client is directly connected to.
-
-The SETTINGS frame precedes any priority signal sent from a client in HTTP/2, so
-a server can determine if it should respect the HTTP/2 scheme before building
-state. A server that receives SETTINGS_DEPRECATE_HTTP2_PRIORITIES with value of
-1 MUST ignore HTTP/2 priority signals.
-
-Where both endpoints disable HTTP/2 priorities, the client is expected to send
-this scheme's priority signal. Handling of omitted signals is described in
+send both the HTTP/2 priority signals and the signals of this prioritization
+scheme (see {{header-field}} and {{h2-update-frame}}). When the client receives
+the first SETTINGS frame that contains the SETTINGS_DEPRECATE_RFC7540_PRIORITIES
+parameter with value of 1, it SHOULD stop sending the HTTP/2 priority signals.
+If the value was 0 or if the settings parameter was absent, it SHOULD stop
+sending PRIORITY_UPDATE frames ({{h2-update-frame}}), but MAY continue sending
+the Priority header field ({{header-field}}), as it is an end-to-end signal that
+might be useful to nodes behind the server that the client is directly connected
+to.
+
+The SETTINGS frame precedes any HTTP/2 priority signal sent from a client, so a
```

Comment:
```suggestion
The SETTINGS frame takes precedence over any HTTP/2 priority signal sent from a client, so a
```
It doesn't necessarily precede it in the pure sense of the word if we're talking about server-sent SETTINGS frame, right?

Reply:
In HTTP/2, SETTINGS has to come before other things because of the preface, so the current text is accurate IMO.